JavaAssignments/src/amarja/Sumequaltonum.java

```diff
@@ -0,0 +1,35 @@
+package amarja;
+//Question 7 Program to find all pair of integer array whose sum is equal to given number
+import java.util.Scanner;
+public class Sumequaltonum {
+	void findpair(int arr[],int no)
```

Comment:
We need to discuss this logic.
Please contact me whenever possible.

Reply:
ok we will disucss on saturday